ddtrace/internal/telemetry/writer.py

```diff
@@ -456,6 +456,7 @@ def _app_started_event(self):
                 ("ddtrace_bootstrapped", config._ddtrace_bootstrapped, "unknown"),
                 ("ddtrace_auto_used", "ddtrace.auto" in sys.modules, "unknown"),
                 ("otel_enabled", config._otel_enabled, "unknown"),
+                ("runtimemetrics_enabled", asbool(os.getenv("DD_RUNTIME_METRICS_ENABLED", default=False)), "unknown"),
```

Comment:
This could be beyond the scope of this PR but should we add a `runtime_metrics` property to the global config object. I think we're moving away from reading environment variables directly. Ideally the global config object should be the only interface for setting and updating configurations. This has the added benefit of "locking in" configurations on application start up (these configurations should only change via RC).

Reply:
I wanted to read from runtime metrics directly but got a circular reference issue.

Adding to `ddtrace.config` would be good, but small issue there is that `runtime_metrics` doesn't use that to check for enabling/disabling, we have the `ddtrace.runtime.RuntimeMetrics.enable()` API. 🤔